src/callback/class-base-ajax-callback.php

```diff
@@ -21,7 +21,7 @@
 abstract class Base_Ajax_Callback implements Invokable, Service {
 
 	/**
-	 * Register ajax callback
+	 * Register ajax callback.
```

Comment:
Please remove full stops in all the places you've added it in all the files in summaries.

Reply:
Isn't that part of the WP Coding Standard? 
(just looking for a reason, not that you need to follow it)

Reply:
I'll revert back what I have done with this file. Thanks

Reply:
@joyously docblocks summaries don't need to have a full stop at the end. The descriptions do.